weblate/templates/translate.html

```diff
@@ -98,15 +98,26 @@
     {% if secondary %}
     {% for secondary_unit in secondary %}
     <div class="form-group">
-    <label>{{ secondary_unit.translation.language }}</label>
+    <label>
+      {{ secondary_unit.translation.language }}
+      {% if unit.translation.plural.number == 1 %}
```

Comment:
You should do this based on `unit.is_plural`, this way it's never shown for languages having plural, but current string does not having one.